pkg/network/ebpf/c/protocols/http2/decoding.h

```diff
@@ -117,9 +117,47 @@ static __always_inline void parse_field_indexed(dynamic_table_index_t *dynamic_i
 
 READ_INTO_BUFFER(path, HTTP2_MAX_PATH_LEN, BLK_SIZE)
 
+// get_bucket_index returns the bucket index for the given path size.
+static __always_inline __u32 get_bucket_index(__u32 size) {
```

Comment:
get_bucket_index is being call with `str_len` parameter which is `__u64`

Reply:
correct nice catch, I have changed it in the function which union both of the functions `update_path_size_telemetry`